### ISSUE TITLE
Unnecessary errors from normal use

### ISSUE THREAD

**Issue by dogerish** [Player11132/The-thick-bot#1]
Whenever I use a command wrong or try to use a command that it doesn't know, it throws a non-fatal exception. Not a terrible thing, but is quite unnecessary. For example, when I gave `imdb` no arguments, it gave no feedback to me on discord, and on the terminal I later saw that it threw an exception:
```
Ignoring exception in command imdb:
Traceback (most recent call last):
  File "/home/pi/.local/lib/python3.7/site-packages/discord/ext/commands/bot.py", line 902, in invoke
    await ctx.command.invoke(ctx)
  File "/home/pi/.local/lib/python3.7/site-packages/discord/ext/commands/core.py", line 856, in invoke
    await self.prepare(ctx)
  File "/home/pi/.local/lib/python3.7/site-packages/discord/ext/commands/core.py", line 790, in prepare
    await self._parse_arguments(ctx)
  File "/home/pi/.local/lib/python3.7/site-packages/discord/ext/commands/core.py", line 706, in _parse_arguments
    kwargs[name] = await self.transform(ctx, param)
  File "/home/pi/.local/lib/python3.7/site-packages/discord/ext/commands/core.py", line 542, in transform
    raise MissingRequiredArgument(param)
discord.ext.commands.errors.MissingRequiredArgument: keyword is a required argument that is missing.
```
It should give feedback to the user instead of throwing an exception.

## TL;DR
It errors whenever a user makes a mistake and doesn't tell the user about that.
